testdata/project-v3/config/manager/manager.yaml

```diff
@@ -33,6 +33,14 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
+        livenessProbe:
+          httpGet:
+            path: /readyz
+            port: 9443
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 9443
```

Comment:
And then here for the readinessProbe:

initialDelaySeconds: 5
      periodSeconds: 10